pdns/dnsdistdist/docs/advanced/ebpf.rst

```diff
@@ -73,3 +73,6 @@ Since 1.6.0, the default BPF filter set via :func:`setDefaultBPFFilter` will aut
 That feature might require an increase of the memory limit associated to a socket, via the sysctl setting ``net.core.optmem_max``.
 When attaching an eBPF program to a socket, the size of the program is checked against this limit, and the default value might not be enough.
 Large map sizes might also require an increase of ``RLIMIT_MEMLOCK``, which can be done by adding ``LimitMEMLOCK=infinity`` in the systemd unit file.
+
```

Comment:
I don't like infinity limits.

Reply:
We are not going ahead with infinity limit

Reply:
`infinity` is still mentioned on line 75.

Also: there is a unit confusion here: the code in `bpf-filter.cc` uses 1 mega byte (`setrlimit`'s unit is bytes), while the `/security/limits.conf` example set the limit to 1GB (it uses 1k byte as unit).
Note that `LimitMEMLOCK` in a systemd unit file uses *bytes* as unit.